apt-hook/json-hook-src/json-hook_test.go

```diff
@@ -14,33 +14,33 @@ func TestCreateUpdateMessages(t *testing.T) {
 		expectedMessage       string
 	}
 	testParamsList := []params{
-		params{0, 0, 0, ""},
```

Comment:
any change here besides dropping "params"

Reply:
Just dropped "params" (I installed a go linter and my editor was yelling at me about it, so I thought I'd squeeze it in here)